diff --git a/node_modules/say/index.d.ts b/node_modules/say/index.d.ts
index c3b6323..86b4fbf 100644
--- a/node_modules/say/index.d.ts
+++ b/node_modules/say/index.d.ts
@@ -7,7 +7,7 @@ declare module 'say' {
     class Say {
       public export(text: string, voice?: string, speed?: number, filePath?: string, callback?: errorCallback): void;
       public speak(text: string, voice?: string, speed?: number, callback?: errorCallback): void;
-      public stop(): void;
+      public stop(callback: (err:any?) => void): void;
       public getInstalledVoices(callback: errorCallback): void;
     }
   }
